src/cargo/util/lev_distance.rs

```diff
@@ -1,93 +0,0 @@
-use std::cmp;
-
-pub fn lev_distance(me: &str, t: &str) -> usize {
```

Comment:
In the future, I'd recommend doing the rename and algorithm changes in separate commits so it is easier to review